editors/code/package.json

```diff
@@ -1455,6 +1455,11 @@
                         "Search in current workspace and dependencies."
                     ]
                 },
+                "rust-analyzer.autoTriggerParameterHints": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "Enable or disable automatic triggering of parameter hints when accepting suggestions."
+                },
```

Comment:
There is no need to add a new config here, we should honor VSCode's native `editor.parameterHints.enabled` config for this

Reply:
I replaced it